tasks.py

```diff
@@ -173,6 +175,34 @@ def clean_all(context):
     pass
 namespace_clean.add_task(clean_all, 'all')
 
+@invoke.task
+def tag(context, name='', message=''):
+    "Add a Git tag and push it to origin"
+    # If a tag was provided on the command-line, then add a Git tag and push it to origin
+    if name:
+        context.run('git tag -a {} -m {!r}'.format(name, message))
+        context.run('git push origin {}'.format(name))
+namespace.add_task(tag)
+
+@invoke.task()
+def validatetag(context):
+    "Check to make sure that a tag exists for the current HEAD and it looks like a valid version number"
+    # Validate that a Git tag exists for the current commit HEAD
+    result = context.run("git describe --exact-match --tags $(git log -n1 --pretty='%h')")
+    tag = result.stdout.rstrip()
+
+    # Validate that the Git tag appears to be a valid version number
+    ver_regex = re.compile('(\d+)\.(\d+)\.(\d+)')
```

Comment:
Do you think we should be more or less strict when it comes to validating that the Git Tag appears to be a valid version number for publishing to PyPI?